docs/mobile/events.md

```diff
@@ -21,46 +20,48 @@ application lifecycle. This event is meant to be used in conjunction with
 `os.name` [resource semantic convention](/docs/resource/os.md) to identify the
 mobile operating system (e.g. Android, iOS).
 
-### iOS
+Payload field MUST be used to describe the state of the application at the
+time of the event. The following table describes the payload fields that MUST
+be used to describe the state of the application at the time of the event.
 
-<!-- semconv ios.lifecycle.events -->
-The event name MUST be `device.app.lifecycle`.
-
-| Attribute  | Type | Description  | Examples  | Requirement Level |
-|---|---|---|---|---|
-| `ios.state` | string | This attribute represents the state the application has transitioned into at the occurrence of the event. [1] | `active` | Required |
+The `android.state` and `ios.state` fields are mutually exclusive and MUST
+NOT be used together, each field MUST be used with its corresponding
+`os.name` [resource semantic convention](/docs/resource/os.md) value.
```

Comment:
Is it still needed?

Reply:
I'm thinking of leaving this for now (as part of this PR) as it provides a link to the OS semantic conventions and highlights it a bit more than just the individual requirement level (at least for now).

Once we have the build tools defined and available this can be reviewed again then.